### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by 101arrowz**
Removed a few dependencies, improved code readability. These are the only issues I could easily resolve with only a cursory look at the codebase.

### CODE REVIEW

File: start.js [hunk 1]
```diff
@@ -4,12 +4,14 @@ const { dependencies } = require('./package');
...
+const PORT = process.env.PORT || 90;
+
 // Giving the launch data and start listening
-app.listen(90, function() {
+app.listen(PORT, function() {
     // I know that this can all go into one log but I prefer to log per line to keep it clean
     console.log("\x1b[34mApp started\x1b[0m");
-    console.log("\x1b[33mModules loaded:\n" + JSON.stringify(dependencies, null, 2) + "\x1b[0m")
-    console.log("\x1b[32mserver address: http://localhost:90\x1b[0m");
+    console.log(`\x1b[33mModules loaded:\n${JSON.stringify(dependencies, null, 2)}\x1b[0m`)
```
Comment (SMJSProductions) [inline start.js:13]: I personally don't like this method of adding variables to a string

File: utils/default values/songs.js [hunk 2]
```diff
@@ -1,90 +1,48 @@
+const songIDs = [
```
Comment (SMJSProductions) [inline utils/default values/songs.js:1]: Was actually planning to do something like this but in the module exporting as json

File: utils/default values/songs.js [hunk 3]
```diff
@@ -1,90 +1,48 @@
...
-            return "Power Trip";
-        default:
-            return "Stereo Madness";
-    }
+  return songIDs[songID] || songIDs[0];
 }
-
-module.exports = init;
\ No newline at end of file
+module.exports = init;
```
Comment (SMJSProductions) [inline utils/default values/songs.js:48]: not sure what got changed here?

File: start.js [hunk 4]
```diff
@@ -4,12 +4,14 @@ const { dependencies } = require('./package');
 const bodyParser = require('body-parser');
 const apihead = require('./headers/API');
 
+const PORT = process.env.PORT || 90;
```
Comment (SMJSProductions) [inline start.js:7]: I was going to make a config json file for this type of stuff so this won't be needed

**Review by SMJSProductions — CHANGES_REQUESTED** [review]
Ok
I'm not going to use anything from start.js since I had other cleanup plans for that

I will use your method of XOR and fix the Base64 mistake I made but the gzip change your suggested will not work due to the buffer type pako returns

File: utils/default values/songs.js (hunk 3 above)
Comment (101arrowz) [reply utils/default values/songs.js:48]: Not sure either. Don't know why it marked it as changed.

File: start.js (hunk 4 above)
Comment (101arrowz) [reply start.js:7]: Eh, environment variables are useful too. I'd still go with something like this.

File: start.js (hunk 1 above)
Comment (101arrowz) [reply start.js:13]: Why not? I find template literals a lot cleaner to read. (Of course, it's still your call to make.)

**Comment by SMJSProductions** [comment]
Ok I found out some things were caused by my outdated node version but I implemented a lot of stuff manually my way so I will close this since I don't need it anymore

I did take a lot as reference so I will put you in the credits

**Pull request closed without merging**